### PR TITLE
Fix tests failing on buster

### PR DESCRIPTION
**Pull request by jooola**
The CI pipeline discovered that we have some inconsistency across distribution with some tests.

For now, we could use some workaround until we properly fix the code.

### CODE REVIEW

File: python_apps/airtime_analyzer/tests/cuepoint_analyzer_test.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import lsb_release
```
Comment (paddatrapper) [inline python_apps/airtime_analyzer/tests/cuepoint_analyzer_test.py:1]: Module not found error. Think uou need python3- packages and added to requirements-dev.txt

Comment (paddatrapper) [reply python_apps/airtime_analyzer/tests/cuepoint_analyzer_test.py:1]: Or use platform https://docs.python.org/3/library/platform.html#platform.uname

Comment (jooola) [reply python_apps/airtime_analyzer/tests/cuepoint_analyzer_test.py:1]: I'll use `distro`, I really didn't think this though before using it. `lsb_release` is a bad idea.

**Comment by jooola** [comment]
Should be good to go !

**Pull request merged**